0-bootstrap/modules/jenkins/main.tf

```diff
@@ -37,13 +42,20 @@ module "cicd_project" {
   labels                      = var.project_labels
 }
 
+resource "google_project_service" "jenkins_apis" {
```

Comment:
you can use `activate_apis ` with `cicd_project` PF.

Reply:
`google_project_service.jenkins_apis` is used in multiple places, so it makes sense to keep it as a separate resource.

Reply:
@caleonardo I am not sure I follow. All the places that use `google_project_service.jenkins_apis` are to enforce explicit dependecy. If you use `activate_apis` this is also not necessary as it will be enforced implicitly.

Reply:
FYI this is separate in the bootstrap module like this, because we had a race condition at the time with APIs not being enabled before we created repos etc. Not sure if this is still the case in newer versions of the project factory

Reply:
Got it. Fixed in jenkins-agent-3 https://github.com/terraform-google-modules/terraform-example-foundation/pull/78

Reply:
@rjerrems I dont think there is any race condition now.

Reply:
This is already fixed in PR 78 which is the valid one. this PR77 is closed now.

Thank you!